packages/fxa-payments-server/src/lib/config.ts

```diff
@@ -64,15 +72,19 @@ function decodeConfig(content: string | null) {
   }
 }
 
-export function readConfigFromMeta() {
-  const configEl = document.head.querySelector('meta[name="fxa-config"]');
+// Define a minimal function type for accessing meta content that's easier to
+// mock, yet still matches real DOM.
+type getMetaElementType = (
+  name: string
+) => null | { getAttribute: (name: string) => null | string };
+
+export function readConfigFromMeta(getMetaElement: getMetaElementType) {
```

Comment:
Maybe move the function from the call site to here as a default param?

Reply:
I'm trying to keep the hard-to-mock DOM APIs out of this file. When I add that back in as a default param, that becomes a line I need to test for coverage. Figured I'd try to keep most / all of that in `index.tsx` and test that all in one spot